data_management/data_factory/models/df3/5_exposition/df3_chantier.sql

```diff
@@ -175,9 +188,39 @@ SELECT
         WHEN
             UPPER(mc.replicate_val_nat_to) = 'REG' AND z.zone_type = 'REG'
             THEN 'reg'::maille
-    END AS donnees_maille_source
+    END AS donnees_maille_source,
+    CASE
+        WHEN
+            ta_ch_today.tag_ch IS NULL
+            THEN NULL::type_tendance
+        WHEN
+            ta_ch_prev_month.tag_ch IS NULL OR ta_ch_today.tag_ch = ta_ch_prev_month.tag_ch
+            THEN 'STAGNATION'::type_tendance
+        WHEN
+            ta_ch_today.tag_ch > ta_ch_prev_month.tag_ch
+            THEN 'HAUSSE'::type_tendance
+        WHEN
+            ta_ch_today.tag_ch < ta_ch_prev_month.tag_ch
+            THEN 'BAISSE'::type_tendance
+        END AS tendance,
+    CASE
+        WHEN
+            ta_ch_today.tag_ch IS NULL
+            THEN NULL
+        WHEN
+            ta_ch_prev_month.tag_ch IS NULL OR ta_ch_today.tag_ch = ta_ch_prev_month.tag_ch
+            THEN 0
+        WHEN
+            ta_ch_today.tag_ch > ta_ch_prev_month.tag_ch
+            THEN 1
+        WHEN
+            ta_ch_today.tag_ch < ta_ch_prev_month.tag_ch
+            THEN -1
+        END AS tendance_ecart,
```

Comment:
A quoi correspond 1 et -1 ?

Reply:
C'est pour définir si l'écart est positif négatif ou stable, ca permet après derrière de trier correctement direct en sql :) 
On peut ajouter de la docs dessus si jamais

Reply:
Genre on a Baisse Hausse Stagnation mais trié par ASC ou DESC ca donne Baisse Hausse Stagnation alors qu'on veut nous Hausse Stagnation Baisse ou inversement d'où la nécessité de revenir à une valeur plus direct. 

Et pourquoi pas avoir la différence réelle, c'est que si 2 écarts sont similaire  (Hausse et Hausse) on veut pouvoir trier après par ID, ce qui n'est pas possible si on a Hausse -> 2.3 et Hausse -> 2.1 par exemple, il considère les écarts comme différent et donc n'a pas besoin de trier par ID. 

Alors que nous la on a Hausse -> 1 et Hausse -> 1 du coup ca applique bien le tri secondaire par ID

Reply:
Un peu mix and twist de mettre de la logique de tri front ici mais je vois l'intérêt.
Dans ce cas, on peut renommer `tendance_ecart -> tendance_int_index` ou un truc comme ça pq je comprends pas ce que fait `ecart` dans ce nom

Reply:
Yes le 'écart' est un rebut de nom du fait qu'au début on avait reelement mis l'écart mais on l'a fait évoluer sans changer le nom, je fix ca en 2-2